tsnex/tsne.py

```diff
@@ -84,6 +85,7 @@ def transform(
     n_iter: int = 1000,
     metric_fn: Callable = euclidean_distance,
     early_exageration: float = 12.0,
+    batch_size: int = None,
```

Comment:
Could you add the `Optional[int]` typing, since we allow for `None` to be passed?